### ISSUE TITLE
Preserve documentation for previous versions

### ISSUE THREAD

**Issue by kantord** [kiwicom/orbit-components#234]
From time to time we have to see the documentation for previous versions, however there's currently no convenient way to do that on the storybook

Make sure these boxes are ticked before submitting a new issue:

* [x] I have checked that this is not a duplicate
* [x] I have added a label categorizing this under BLKN | Kiwicom | Icons | Global

**Comment by darkwindcz** [comment]
For what component did it happen pls? We are trying to update & sync docs currently but maybe we missed something somewhere

**Comment by kantord** [comment]
We'd like to see old documentation for previous versions in general. For example if we are depending on rc9 at the moment, rc12 documentation is not useful for us, as the APIs changed

**Comment by Fallion** [comment]
@kantord Isn't the solution to simply switch to the tag?

**Comment by Fallion** [comment]
<img width="164" alt="screen shot 2018-06-27 at 23 32 54" src="https://user-images.githubusercontent.com/683286/42000885-73c2a166-7a62-11e8-9902-4a1fc536553f.png">

**Comment by kantord** [comment]
> Or are you talking about storybook?

Yep, storybook.

**Comment by wEprous** [comment]
Storybook will be preserved with staging from now. We will be posting links to changelog and also on Slack.
